bindata/v3.11.0/openshift-controller-manager/ds.yaml

```diff
@@ -55,8 +57,13 @@ spec:
       - name: serving-cert
         secret:
           secretName: serving-cert
+      - name: proxy-ca-bundles
+        configMap:
+          name: openshift-global-ca
```

Comment:
but rather than the local namespace CM we copy in, we use our injected CM source in the OCM namesapce, `openshift-global-ca`